### ISSUE TITLE
S3DIS_segmentation.ipynbのlossの計算について

### ISSUE THREAD

**Issue by TaroKawa** [3d-point-cloud-processing/3dpcp_book_codes#6]
S3DIS_segmentation.ipynbにおいて、lossの計算をする際、

現状
loss = class_loss + reg_loss * 0.001 * 0.001

となっておりますが、
loss = class_loss + reg_loss * 0.001
ではないでしょうか？

modelnet10_classification.ipynbの中では、0.001になっています。
